website/docs/r/extension.html.markdown

```diff
@@ -71,7 +71,8 @@ EOF
 The following arguments are supported:
 
   * `name` - (Optional) The name of the service extension.
-  * `endpoint_url` - (Optional) The url of the extension.
+  * `endpoint_url` - (Reqeuired|Optional) The url of the extension.  
```

Comment:
Can you fix the spelling on Required here?

Reply:
What about leaving it `Optional` as that's how it is defined on the terraform schema, so terraform does not enforce it, but being more explicit about some exceptions for `Generic V2 Webhook` like described in https://github.com/terraform-providers/terraform-provider-pagerduty/issues/131#issuecomment-489375555 ?

Reply:
22cd0e6602247e99626af8afd405d3e10f60c018 fixed the typo @stmcallister

Reply:
I like the additional note! Nice job!